about.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>About</title>
+  <link rel="stylesheet" href="about.css">
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato:&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header class="first-header">
+    <a href=""><img src="imgs/facebook.png" alt=""></a>
+    <a href=""><img src="imgs/twitter.png" alt=""></a>
+    <p>English</p>
+    <p>My Page</p>
+    <p>Logout</p>
+  </header>
+    <header class="headr">
+    <nav class="nav-class">
+      <div class="ham_menu_btn">
+        <a href="" id="ham_menu_button">
+          <img src="imgs/mobile_menu_btn.png" alt="hamburger menu">
+        </a>
+      </div>
+      <ul class="mobile-nav">
+        <li class="mobile-item"><a href="#" id="close_mobile_menu"><img id="nav-close-btn" src="imgs/close_mobile_menu.png" alt="close button"></a>
+        </li>
+        <li class="mobile-item"><a href="index.html">Home</a></li>
+        <li class="mobile-item"><a href="about.html">About</a></li>
+        <li class="mobile-item">Join</li>
+        <li class="mobile-item" id="pres"><a href="#">Presentation</a></li>
+      </ul>
+      <ul class="desktop_menu">
+        <li class="menu_item intro"><a href="about.html">About</a></li>
+        <li class="menu_item program">Program</li>
+        <li class="menu_item application">Join</li>
+        <li class="menu_item sponsor">Sponsor</li>
+        <li class="menu_item news">News</li>
+        <li class="menu_item presentation">
```

Comment:
- [ ] the desktop navbar looks awesome, kindly try to reduce the spaces between the links so they look correctly aligned to the right as shown in the template 

![image](https://user-images.githubusercontent.com/89282221/190811331-14157c9f-09cc-4f45-9c5a-4fb86cf8dc0b.png)